content/docs/concepts/what-is-istio/index.md

```diff
@@ -73,9 +73,9 @@ secure pod-to-pod or service-to-service communication at the network and applica
 
 ### Observability
 
-Istio’s robust tracing, monitoring, and logging give you deep insights into your service mesh deployment. Gain a real understanding of how service performance
+Istio’s robust tracing, monitoring, and logging features give you deep insights into your service mesh deployment. Gain a real understanding of how service performance
 impacts things upstream and downstream with Istio’s monitoring features, while its custom dashboards provide visibility into the performance of all your
-services and let you see how that performance is affecting your other processes.
+services and lets you see how that performance is affecting your other processes.
```

Comment:
Dashboards is plural, so I think this should just be "let". Correct?

Reply:
you are right actually